models/datastore_test.go

```diff
@@ -0,0 +1,113 @@
+package models_test
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/atmiguel/cerealnotes/models"
+	"github.com/atmiguel/cerealnotes/test_util"
+)
+
+var postgresUrl = "postgresql://localhost/cerealnotes_test?sslmode=disable"
+
+const noteTable = "note"
+const publicationTable = "publication"
+const noteToPublicationTable = "note_to_publication_relationship"
+const noteToCategoryTable = "note_to_category_relationship"
+const userTable = "app_user"
+
+var tables = []string{
+	noteToPublicationTable,
+	publicationTable,
+	noteToCategoryTable,
+	noteTable,
+	userTable,
+}
+
+func ClearAllValuesInTable(db *models.DB) {
```

Comment:
why ? isn't the how they are named now more descriptive?

Reply:
I don't think so. As it reads right now it sounds like it's clearing a single table, but it's clearing multiple tables. 

While we're on the topic,  I also think the phrase AllValues is superfluous. I'd prefer the first function be named ClearTable and the second ClearTables/ClearAllTables/ClearDatabase.

Also also, consider using the word Truncate instead of Clear. That word carries more meaning since it's the actual sql operation you're doing. I'm not sure I agree with using truncate, there's an argument to be made that using the word truncate leaks too much implementation detail to the caller. But it's a thought worth considering.